### ISSUE TITLE
v0.36.0-old

### ISSUE THREAD

**Issue by philknows** [ChainSafe/lodestar#3748]
This issue tracks significant inclusions and status of fixes + features of the next upcoming release:

| Problem (Issue)  | Solutions (PR)          | Urgency  | Merged? | Impact/Notes |
| ---------------- | ------------------- | -------- | --------- | ------------- |
| #3548 | #3661 | High | Yes | Improvements to Lodestar: peer scoring cache, fast msg id, etc. |
| #3523 | #3522 | Medium |  No |  Enable users to manage keystores |
| #3688 | #3753 | High | Yes |  For Wagyu Installer: Allow to pass password as a file path, and ensure it's not interactive.  | 
| #3428 |             | High | Closed |  The next sync committee branch should be created against the post-state of the finalized checkpoint block.  | 
| #3495 |             | High | Yes |   The light-client server should not prepare updates during sync  |
| #3732 | #3738 | High | Yes | Fix for performance regression |
| #3749 | #3745  | High | Yes | Fix for performance regression |
| N/A | #3595 | High | Yes | Data loss from Circular Reference objects being dumped |
|  | #3760  | High | No | SSZ Upgrade |

Linked milestone https://github.com/ChainSafe/lodestar/milestone/13

**Comment by philknows** [comment]
Bumped this tracker from `0.35.0` to `0.36.0` release. 
Decision was made to push `0.35.0` earlier without these planned merges to fix important bug described in #3700 persisting in `0.34.0`.

**Comment by philknows** [comment]
Tasks in this old release milestone are complete, but blocked on other issues for a stable release. Closing in favour of #3884.
